CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+## v1.16.0 (13/06/2023)
+
+**Common**
+* Added `olp::http::CertificateSettings` struct to store custom certificate settings.
+* Added `olp::http::NetworkInitializationSettings` struct containing `olp::http::CertificateSettings` to be passed to `olp::http::CreateDefaultNetwork(..)`.
+* Deprecated `olp::http::CreateDefaultNetwork()` use added `olp::http::CreateDefaultNetwork(..)` that take `olp::http::NetworkInitializationSettings`.
+* Added `olp::client::CreateDefaultNetworkRequestHandler(..)` that take `olp::http::NetworkInitializationSettings` as an argument.
+* Extended `olp::cache::CacheSettings` with `extend_permissions` option.
+* Extended `olp::http::NetworkSettings` with `GetMaxConnectionLifetime()` and `WithMaxConnectionLifetime(..)`.
+* Deprecated `GetRetries()` and `WithRetries(..)` in `olp::http::NetworkSettings`.
+* Extended `olp::http::NetworkSettings` with `GetConnectionTimeoutDuration()`, `WithConnectionTimeout(std::chrono::milliseconds timeout)`, `GetTransferTimeoutDuration()` and `WithTransferTimeout(std::chrono::milliseconds timeout)`.
+* Deprecated `GetConnectionTimeout()`, `WithConnectionTimeout(int timeout)`, `GetTransferTimeout()` and `WithTransferTimeout(int timeout)` in `olp::http::NetworkSettings`. Use milliseconds instead.
+* Required TLS 1.2 or later for network connection.
+* Fixed config failure when CMAKE_BUILD_TYPE not set.
```

Comment:
Please specify that this is a CMake configuration

Reply:
Done